transactional_licence_form/forms.py

```diff
@@ -0,0 +1,317 @@
+
+
+def with_label_class(class_name, field):
+    field.label_class = class_name
+    return field
+
+
+def list_to_choices(values):
+    return [(v, v) for v in values]
+
+
+class FCLFormMixin(object):
+    def __init__(self, **kwargs):
+        self.send_to_dynamics = kwargs.pop("send_to_dynamics", True)
+        self.label_class = kwargs.pop("label_class", None)
+        super(FCLFormMixin, self).__init__(**kwargs)
+
+
+class FCLCharField(FCLFormMixin, forms.CharField):
+    pass
+
+
+class FCLChoiceField(FCLFormMixin, forms.ChoiceField):
+    pass
+
+
+class FCLMultipleChoiceField(FCLFormMixin, forms.MultipleChoiceField):
+    pass
+
+
+class ContactForm(forms.Form):
+
+    ALTERNATIVE_CONTACT_CHOICES = [
+        "This is the same person as the main contact",
+        "This is a different person (please enter their details below)",
+    ]
+
+    contact_lastname = FCLCharField(
+        label="The full name of the person we can discuss your licence application with",
+        max_length=50,
+    )
+    contact_email = FCLCharField(
+        label="The email address of the person we can discuss your licence application with",
+        max_length=50,
+    )
+    alternative_contact = FCLChoiceField(
+        label=(
+            "We need to ensure we have contact details of the person in your "
+            "organisation who will be responsible for licence compliance. This "
+            "should be someone senior who has overall responsibility for "
+            "ensuring your organisation complies with all terms and conditions "
+            "of the licence."
+        ),
+        widget=forms.RadioSelect,
+        choices=list_to_choices(ALTERNATIVE_CONTACT_CHOICES),
+        label_class="as_para",
+    )
+    licence_holder_lastname = FCLCharField(
+        label="The full name of the person in your organisation who will be responsible for licence compliance",
+        max_length=50,
+        show_hidden_initial=True,
+    )
+    licence_holder_email = FCLCharField(
+        label="The email address of the person in your organisation who will be responsible for licence compliance",
+        max_length=50,
+        show_hidden_initial=True,
+    )
+
+
+class OrganizationForm(forms.Form):
+
+    TNA_CONTACTTYPE_CHOICES = [
+        "Private limited company",
+        "Public limited company",
+        "Partnership",
+        "Sole trader",
+        "Registered Charity",
+        "Community interest company",
+        "Independent research organisation",
+        "Public body",
+        "Independent body",
+        "Other (please specify)",
+    ]
+
+    agent_companyname = FCLCharField(
+        label="What is the full legal name of your organisation?", max_length=50
+    )
+    agent_companyname_other = FCLCharField(
+        label="Please enter any other names your organisation is known by",
+        max_length=50,
+        help_text="If your organisation is not known by any other names, please type <strong>none</strong>.",
+    )
```

Comment:
Could we set this field to not being required rather than forcing the user to write html tags?